installers/rpm-code/codes/geant4.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+: notes <<'EOF'
+
+If you see the following errors try setting LIBGL_ALWAYS_INDIRECT=1
+libGL error: No matching fbConfigs or visuals found
+libGL error: failed to load driver: swrast
+https://unix.stackexchange.com/questions/589236/libgl-error-no-matching-fbconfigs-or-visuals-found-glxgears-error-docker-cu
+https://unix.stackexchange.com/questions/1437/what-does-libgl-always-indirect-1-actually-do
+
+To help debug libgl errors:
+LIBGL_DEBUG=verbose
+
+On a Mac if the geant4 GUI appears but the space where the visualization should be is blank you may
+need to enable opengl:
+https://geant4-forum.web.cern.ch/t/qt-viewer-scene-tree-empty/1207/5 (which links to)
+https://services.dartmouth.edu/TDClient/1806/Portal/KB/ArticleDet?ID=89669
+EOF
+
+geant4_install_data_download_script() {
+    install -m 555 /dev/stdin "${codes_dir[bin]}"/rs_geant4_download_data <<'EOF'
+#!/bin/bash
+set -eou pipefail
+
+install_d=${1:-}
+if [[ ! $install_d ]]; then
+    echo "usage: $0 dataset-install-directory
+You must supply a directory where you want the datasets to be installed" 1>&2
+    exit 1
+fi
+
+declare datasets=(
+    'G4EMLOW.8.2'
+    'G4PhotonEvaporation.5.7'
+    'G4RadioactiveDecay.5.6'
+    'G4PARTICLEXS.4.0'
+    'G4PII.1.3'
+    'G4RealSurface.2.2'
+    'G4SAIDDATA.2.0'
+    'G4ABLA.3.1'
+    'G4INCL.1.0'
+    'G4ENSDFSTATE.2.3'
+    'G4TENDL.1.4'
+)
+
+for d in ${datasets[@]}; do
+    echo "Installing: $d"
+    curl -s -S -L https://cern.ch/geant4-data/datasets/"$d".tar.gz | tar xz --directory="$install_d"
+done
+
+echo "
+You must set 'GEANT4_DATA_DIR=$install_d' when running geant4 for it to find the datsets
+For example: GEANT4_DATA_DIR=$install_d ./exampleB1"
```

Comment:
Put this on its own line to make it easier to copy and paste:

```txt
For example: 
GEANT4_DATA_DIR=$install_d ./exampleB1"
```

I find multiline messages are easier to read with `cat <<EOF`, but this is a style thing.